tests/test_coverage.py

```diff
@@ -55,7 +56,8 @@ def test_happy_cases(file_name, object_type):
     (
         "point-series-domain-more-z.json",
         Domain,
-        r"If provided, the 'values' field of the 'z'-axis of a 'PointSeries' domain must contain a single value.",
+        r"If provided, the 'values' field of the ValuesAxis 'z'-axis of a 'PointSeries' " + \
+            "domain must contain a single value.",
```

Comment:
```suggestion
        r"If provided, the 'values' field of the ValuesAxis 'z'-axis of a 'PointSeries' "
        + "domain must contain a single value.",
```
Sorry, black was still unhappy. This should work.